_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,114 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku"
+title:  "Presenting JS-Waku"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
+permalink: /2021-05-21-js-waku
+categories: platform
+summary: "Presenting JS-Waku"
+discuss: 
+---
+
+After almost three months working on JS-Waku,
+we thought it would be a good time to give it a proper introduction.
+
+<!-- Waku v2 is built for Status and Ethereum ecosystem -->
+
+First, let's review what is Waku v2.
+
+Waku v2 comes from a need to have a more scalable, better optimised solution for the Status app to achieve decentralised
+communications on resource restricted devices (ie, phones).
+We see the need of such solution in the broader Ethereum ecosystem, beyond Status.
+This is why are building Waku v2 as a decentralised communication platform for all to use and build on.
+If you want to read more about Waku v2 and what it aims to achieve,
+checkout [What's the Plan for Waku v2?](/waku-v2-plan).
+
+<!-- present waku v2 ecosystem pre js-waku, ie, wakunode2 -->
+
+Since last year, we have been busy defining and implementing Waku v2 protocols in [nim-waku](https://github.com/status-im/nim-waku),
+from which you can build [wakunode2](https://github.com/status-im/nim-waku#wakunode).
+Wakunode2 is an adaptive and modular Waku v2 node,
+it allows users to run their own node and use the Waku v2 protocols they need.
+The nim-waku project doubles as a library, that can be used to add Waku v2 support to native applications.
+
+<!-- present the gap js-waku is trying to fill -->
+
+We believe that dApps and wallets can benefit from the Waku network in several ways.
+For some dApps, it may make sense to enable peer-to-peer communications.
+For others, machine-to-machine communications could be a great asset.
+For example, in the case of a DAO with a governance token,
+Waku could be used for [gas-less polling](https://twitter.com/ethstatus/status/1293963402705018881?s=20).
+Enabling the DAO to notify their users of a new poll, and users to vote without interacting with the blockchain.
+
+<!-- explain how js-waku is filling the gap -->
+
```

Comment:
resource restricted environment, zoom back, browser is one such example but also mostly offlnie etc